llmproxy/models/cohere.py

```diff
@@ -11,6 +11,58 @@
     },
 }
 
+cohere_category_data = {
+    "model-categories": {
+        "command": {
+            "Code Generation Task": 2,
+            "Text Generation Task": 1,
+            "Translation and Multilingual Applications Task": 2,
+            "Natural Language Processing Task": 1,
+            "Conversational AI Task": 1,
+            "Educational Applications Task": 2,
+            "Healthcare and Medical Task": 3,
+            "Legal Task": 3,
+            "Financial Task": 3,
+            "Content Recommendation Task": 2
+        },
+        "command-light": {
+            "Code Generation Task": 3,
+            "Text Generation Task": 2,
+            "Translation and Multilingual Applications Task": 3,
+            "Natural Language Processing Task": 2,
+            "Conversational AI Task": 2,
+            "Educational Applications Task": 3,
+            "Healthcare and Medical Task": 4,
+            "Legal Task": 4,
+            "Financial Task": 4,
+            "Content Recommendation Task": 3
+        },
+        "command-nightly": {
+            "Code Generation Task": 2,
+            "Text Generation Task": 1,
+            "Translation and Multilingual Applications Task": 2,
+            "Natural Language Processing Task": 1,
+            "Conversational AI Task": 1,
+            "Educational Applications Task": 2,
+            "Healthcare and Medical Task": 3,
+            "Legal Task": 3,
+            "Financial Task": 3,
+            "Content Recommendation Task": 2
+        },
+        "command-light-nightly": {
+            "Code Generation Task": 3,
+            "Text Generation Task": 2,
+            "Translation and Multilingual Applications Task": 3,
+            "Natural Language Processing Task": 2,
+            "Conversational AI Task": 2,
+            "Educational Applications Task": 3,
```

Comment:
We need to take this out of the class for sure. We may need to do so before merging

Reply:
Original method was to store it in the yaml but thats currently be refactored. I could make another class that has all the info stored as a hashmap and call that instead? What are your thoughts?